marlowe-playground-client/src/Main.purs

```diff
@@ -31,7 +32,7 @@ ajaxSettings = SPSettings_ $ (settings { decodeJson = decodeJson, encodeJson = e
   where
   SPSettings_ settings = defaultSettings $ SPParams_ { baseURL: "/api/" }
 
-  jsonOptions = defaultOptions { unwrapSingleConstructors = true }
+  jsonOptions = defaultOptions { unwrapSingleConstructors = true, sumEncoding = aesonSumEncoding }
```

Comment:
are we 100% sure about this? Is it tested?

Reply:
I can remove it. I don’t use JSON anymore.